response_operations_ui/templates/create-collection-exercise.html

```diff
@@ -21,76 +23,77 @@ <h2 id="error-creating-ce-message" class="ons-u-fs-r--b">Error creating collecti
-                            "value": period,
-                            "attributes": {
-                                "required": true
-                            }
+                        onsButton({
+                            "text": "Save and continue",
+                            "id": "save-btn",
+                            "classes": 'ons-u-mt-l',
+                            "variants": ['primary'],
+                            "submitType": "timer",
                         })
                     }}
                     {{
-                        onsInput({
-                            "id": "user_description",
-                            "name": "user_description",
-                            "type": "text",
-                            "label": {
-                                "text": 'Period as shown to respondent'
-                            },
-                            "maxLength": 50,
-                            "value": user_description,
-                            "attributes": {
-                                "required": true
-                            }
+                        onsButton({
+                            "text": "Cancel",
+                            "id": "cancel-btn",
+                            "classes": 'ons-u-mt-l',
+                            "variants": ['secondary'],
+                            "url": url_for('surveys_bp.view_survey', short_name=short_name),
+                            "submitType": "timer",
+                            "noIcon": "true"
                         })
-                    }}
-                {% endcall %}
-
-                <div class="ons-grid ons-grid--flex ons-grid--gutterless ons-grid--vertical-center ons-grid--no-wrap@s ons-u-mt-m ons-u-mb-m">
-                    <div class="ons-grid__col">
-                        {{
-                            onsButton({
-                                "text": "Save",
-                                "id": "save-btn"
-                            })
-                        }}
-                    </div>
-                    <div class="ons-grid__col ons-u-ml-m">
-                        <a id="cancel-btn" href="{{ url_for('surveys_bp.view_survey', short_name=short_name, period=period) }}">Cancel</a>
-                    </div>
+                    }} 
                 </div>
             </form>
+            <div class="ons-grid__col ons-col-3@xs">
+                {{
+                    onsAccordion({
+                        "id": "infoAccordion",
+                        "open": true,
+                        "itemsList": [
+                            {
+                                "title": "What is a Period ID",
+                                "content": "<p>The period ID is a two, four or six digit number that represents the month or year that a survey runs until.</p><p>'202212' or '2212' would represent December 2022.</p><p>'22' would represent the year 2022.</p>"
```

Comment:
The text in the accordion needs to be allowed to take up more slots in the grid as it's a bit narrow in comparison to the figma design.  Also, should it be open by default?  That's more of a question for Dina, but I think it's a bit ugly having accordions open by default... at that point you might as well always display it instead of bothering with an accordion